scripts/drmgeneratepatch

```diff
@@ -14,9 +14,9 @@ if [ $# -ne 3 ]; then
 	usage
 fi
 
-LINUX_DIR=$1
+LINUX_DIR=$(cd "$1" && pwd)
```

Comment:
realpath might be better.

Reply:
Probably :-) I'm used to use cd(1) because realpath(1) is not available everywhere (or at least wasn't).

Reply:
Fixed.